pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -37,6 +37,20 @@ stdenv.mkDerivation rec {
       nodePackages.pnpm
     ];
 
+    # NOTE: This requires pnpm 8.10.0 or newer
```

Comment:
maybe we could have an assertion here using `lib.versionAtLeast`? it might be nice for anyone overriding or overlaying the package